docs/basic-features/data-fetching.md

```diff
@@ -660,7 +660,7 @@ export async function getServerSideProps(context) {
 The `context` parameter is an object containing the following keys:
 
 - `params`: If this page uses a dynamic route, `params` contains the route parameters. If the page name is `[id].js` , then `params` will look like `{ id: ... }`. To learn more, take a look at the [Dynamic Routing documentation](/docs/routing/dynamic-routes.md).
-- `req`: [The HTTP IncomingMessage object](https://nodejs.org/api/http.html#http_class_http_incomingmessage).
+- `req`: [The HTTP IncomingMessage object](https://nodejs.org/api/http.html#http_class_http_incomingmessage), with an additional `req.cookies` attribute – an object containing the cookies sent by the request (defaults to `{}`).
```

Comment:
I think we need a more formal definition here.

For example, [API routes](
https://nextjs.org/docs/api-routes/introduction) uses the following:

- req: An instance of http.IncomingMessage, plus some [pre-built middlewares](https://nextjs.org/docs/api-routes/api-middlewares)
- res: An instance of http.ServerResponse, plus some [helper functions](https://nextjs.org/docs/api-routes/response-helpers)

cc @leerob @molebox

Reply:
@styfle yeah good call, I was wondering about that too 🤔